@navikt/ds-icons/stories/icons.stories.tsx

```diff
@@ -1,35 +0,0 @@
-import React from "react";
```

Comment:
Burde vi ikke ha denne for å få bedre tilbakemeldinger fra chromatic? Eller bruker vi kanskje ikke chromatic så aktivt om dagen?

Reply:
Litt usikker på denne.. 
Bra:
- Får endringer opp i chromatic
Dårlig: 
- Storybook takler ikke dynamisk generering av 300+ stories veldig bra
- Icons-stories åpnes av seg selv, noe som gjør at publisert storybook side er litt rar

Tenker kanskje det vil være mer verdi å skrive en god story som viser frem Ikon-komponenten med ett ikon.

Reply:
Her om dagen så jeg noen som hadde separat chromatic-storybook. Hadde det vært en ide?

Reply:
Hørtes ut som mye ekstra kode.. kan prøve å ta i bruk Env-variabler: https://storybook.js.org/docs/react/configure/environment-variables

Eks: `CHROMATIC=true` vil rendre alle ikonene, men ellers bare ett ikon som viser ikon-komponenten

Reply:
Ja, det var lurt. En annen måte å gjøre det samme på kan være å ha et eget chromatic filename pattern. F.eks. `*.chromatic.@(js|jsx|ts|tsx|mdx)`

Reply:
Bare en ide. Vet ikke hva som er beste løsning.